From: Tomas Henzl <thenzl@redhat.com>
Date: Fri, 17 Apr 2009 14:53:32 +0300
Subject: [scsi] MPT fusion: remove annoying debug message
Message-id: 49E86DBC.8060806@redhat.com
O-Subject: [RHEL5.4 PATCH3/3] remove annoying debug message
Bugzilla: 475455
RH-Acked-by: David Milburn <dmilburn@redhat.com>

Resolves bz475455.

This patch removes a debug message which annoyed some customers.
In RHEL5.1 the message was by default off (it depended on debug level),
this is now not easy doable, because for this we need iocp != NULL.
So the message is in fact removed here - DMPT_DEBUG_IOCTL is not used
in our current in our sources.
Not in upstream.

diff --git a/drivers/message/fusion/mptctl.c b/drivers/message/fusion/mptctl.c
index 6a0fb3c..be62e26 100644
--- a/drivers/message/fusion/mptctl.c
+++ b/drivers/message/fusion/mptctl.c
@@ -591,8 +591,10 @@ __mptctl_ioctl(struct file *file, unsigned int cmd, unsigned long arg)
 	iocnumX = khdr.iocnum & 0xFF;
 	if (((iocnum = mpt_verify_adapter(iocnumX, &iocp)) < 0) ||
 	    (iocp == NULL)) {
+#ifdef	DMPT_DEBUG_IOCTL
 		printk(KERN_DEBUG MYNAM "%s::mptctl_ioctl() @%d - ioc%d not found!\n",
 				__FILE__, __LINE__, iocnumX);
+#endif
 		return -ENODEV;
 	}
 
